VideoWeb/VideoWeb/ClientApp/src/app/waiting-space/participant-waiting-room/participant-waiting-room.component.html

```diff
@@ -42,24 +42,38 @@
       Your browser does not support Video
     </video>
 
-    <button
-      id="muteUnmuteButton"
-      class="govuk-button reduce-margin"
-      data-module="govuk-button"
-      type="button"
-      (click)="muteUnmuteCall()"
-      aria-label="Mute Microphone Button"
-      [ngClass]="audioMuted ? 'govuk-button--warning' : 'govuk-button'"
-    >
-      {{ audioMuted ? 'Unmute' : 'Mute' }}
-
-      <span *ngIf="!audioMuted" class="govuk-button__start-icon">
-        <img id="mute-microphone-img" class="microphoneIcon" src="/assets/images/mic_on.png" alt="Mute Microphone Image" />
-      </span>
-      <span *ngIf="audioMuted" class="govuk-button__start-icon microphoneIcon">
-        <img id="unmute-microphone-img" class="microphoneIcon" src="/assets/images/mic_off.png" alt="Unmute Microphone Image" />
-      </span>
-    </button>
+    <span *ngIf="!audioMuted">
```

Comment:
Please use ngIfElse here

Reply:
What is advantage?

Reply:
it's a better use of the framework and reduces maintenance and complexity